exercises/practice/queen-attack/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "authors": [
+    "habere-et-dispertire"
+  ],
+  "files": {
+    "solution": [
+      "QueenAttack.rakumod"
```

Comment:
The module names in this file need changing for the configlet workflow to pass.

Reply:
What about the test file reference ? Do we rename that too ?

~~~json
"test": [
      "queen-attack.rakutest"
    ],
~~~

Reply:
No. The test files are always the name of the exercise itself. Packages however are named as they might best fit (the default being camel-case of the exercise name), as sometimes I feel the names of the exercises make for unusual names for the packages themselves. In this instance, I asked for the package to be named `Queen` because the package is intended to be an object which represents the chess queen piece.